wcivf/apps/elections/models.py

```diff
@@ -216,6 +217,16 @@ def ynr_link(self):
             settings.YNR_UTM_QUERY_STRING,
         )
 
+    @property
+    def short_cancelled_message(self):
```

Comment:
Just to make it clear what this does, can you call it `short_cancelled_message_html` or something that indicates it (might) output HTML?

I worry with this pattern (not new in this PR) that we introduce a catch all `mark_safe` and use it in templates when future refactoring might end up taking some sort of query string or use input and opening us up to an XSS. 

Appending "html" to a method name doesn't stop this of course, but it might make us think a little about how it's used.

Reply:
Have changed the name as suggested. In terms of how we handle this in general, as far as I can see there are a few ways this can work:

* run `mark_safe()` on some text in the model/view layer
* don't do that but render it with the `|safe` filter in the template layer
* make small template fragments which you can `{% include %}` in lots of places

If you reckon one of the others is a better way of doing it, I can revise. Regardless of whether you use `mark_safe()` or `|safe` as long as we're calling a model method/`@property` from inside a template, I don't think we can pass in any user input. In order to add a param we'd have to convert to calling in the view with a param and then passing to the template in the context object.

Reply:
Yep, that's a good point about it coming from the model. In general `|safe` or a `*_html` method is something that tells me "this is unsafe", where as typing `foo.no_worries_mate` wouldn't. Th more things we can do to trigger that thought the better – I think this is a good pattern in this case. 

I already approved the PR so I can't do that again, other than to say 👍